intranet/templates/polls/poll.html

```diff
@@ -40,10 +40,15 @@ <h3>
         for
         {% for group in poll.groups.all %}
             {{ group }}
-            {% if not forloop.last %},{% endif %}
```

Comment:
The bullets on line 43 & 47 should not be placed here in the logic, since, if the user is not a polls admin, there is a stray bullet at the end. The bullet should likely be placed on line 50.